lib/fast_jsonapi/object_serializer.rb

```diff
@@ -235,6 +237,25 @@ def fetch_polymorphic_option(options)
         return option if option.respond_to? :keys
         {}
       end
+
+      def get_serializer(item)
+        unless item.to_s.include?('.')
```

Comment:
The logic seems very similar for both parts of this method. Is there anyway we can combine them and make it more concise?